src/metabase/api/automagic_dashboards.clj

```diff
@@ -143,17 +144,31 @@
   [show]
   (cond-> show (= "all" show) keyword))
 
+(defn get-automagic-dashboard
+  "Return an automagic dashboard for entity `entity` with id `id`."
+  [entity entity-id-or-query show]
```

Comment:
It's be great if the docstring explained what role `show` plays. (I know it was undocumented in the original too, so if it's difficult to explain properly, skip it.)